bundles/org.openhab.binding.ws980wifi/README.md

```diff
@@ -0,0 +1,92 @@
+# ws980wifi Binding
+
+The ws980wifi binding connects openHAB with ELV WS980WIFI Weather Stations (see de.elv.com).
+It discovers WS980WIFI Weather Stations in the local network and polls the actual data on a regular base.
+The WS980WIFI has to be part of the local network via WiFi integration.
+Only the actual measures are supported, no historical data.
+The binding reads the data of the station, but does not support to write data back (e.g. updates, configuration parameters).
+
+## Supported Things
+
+ELV WS980WIFI Weather Station.
+
+## Discovery
+
+As soon as the WS980WIFI Weather Station is integrated into your local network, this binding is able to discover the weather station automatically and provides  21 measures as channels of the discovered thing. 
+
+## Binding Configuration
+
+The binding provides no configuration parameters.
+
+## Thing Configuration
+
+Discovered things have one non advanced configuration parameter, which can be modified. 
+It is the Refresh Interval. Default is 60 seconds, min. is 30 seconds.
+The advanced configuration parameters should not be modified.
+
+## Channels
+
+| Channel | Type   | Read/Write | Description                 |
+|---------|--------|------------|-----------------------------|
+| tempInside | Number | R | Temperature Inside in °C |
```

Comment:
Use correct number type : `Number: Temperature`. Also for other channels and in the example this should also be used.